src/nrniv/netpar.cpp

```diff
@@ -79,16 +79,6 @@ extern double nrnmpi_step_wait_; // barrier at beginning of spike exchange.
  */
 extern "C" int nrnthread_all_spike_vectors_return(std::vector<double>& spiketvec, std::vector<int>& spikegidvec);
 
-// BGPDMA can be 0 or 1
-// (BGPDMA & 1) > 0 means multisend ISend allowed
-#if !defined(BGPDMA)
-#define BGPDMA 0
-#endif
-
-#if BGPDMA == 0
-double nrn_bgp_receive_time(int) { return 0.; }
-#endif
```

Comment:
Not sure about this one, because if I look at the nrn_bgp_receive_time (the other one) it seems that it handles bgpdma == 0 already. We should discuss it.

Reply:
@nrnhines : could you comment on this? there is another implementation in `nrniv/bgpdma.cpp`.